helpers/auth/authentication.py

```diff
@@ -91,7 +93,7 @@ def save_user(self):
             pass
         return True
 
-    def user_roles(self, *expected_args):
+    def user_roles(self, *expected_args):  # noqa: C901
```

Comment:
Hey @michael-basweti, Great work. Kindly explain the reason for this `noqa`

Reply:
The function exceeded the functionality complex required, in this context, I would not reduce the complexity and maintain the functionality.